mlflow/types/chat.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import annotations
+
+from typing import Literal, Optional, Union
+
+from pydantic import BaseModel as _BaseModel
+from pydantic import Field
+from typing_extensions import Annotated
+
+from mlflow.utils import IS_PYDANTIC_V2_OR_NEWER
+
+
+class BaseModel(_BaseModel):
+    @classmethod
+    def validate(cls, obj: any):
```

Comment:
```suggestion
    def validate(cls, obj: Any):
```

Interesting. This should be detected by our custom linter.